.github/workflows/update-docs-manual.yml

```diff
@@ -0,0 +1,113 @@
+name: Manual Update MeshLib Documentation Site
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      output_folder:
+        required: false
+        type: string
+        default: "MeshLib/dev"
+  
+jobs:
+  create-stubs:
+    timeout-minutes: 40
+    runs-on: [self-hosted, linux-arm64]
+    container:
+      image: meshlib/meshlib-ubuntu22-arm64:latest
+      options: --user root
+    strategy:
+      fail-fast: false
+    env:
+      CONFIG: Release
+      CXX-COMPILER: /usr/bin/clang++-14
+      CXX-STANDARD: 20
```

Comment:
Probably it would be more optimal to use Ubuntu 24.04 and Clang 18 as mrbind requires Clang 18.